dsc/workflows/base/__init__.py

```diff
@@ -631,7 +575,11 @@ def send_report(
             subject=report.subject,
             source_email_address=CONFIG.source_email,
             recipient_email_addresses=email_recipients,
-            message_body_plain_text=report.to_plain_text(),
-            message_body_html=report.to_html(),
-            attachments=report.create_attachments(),
+            message_body=report.generate_summary(),
+            attachments=[
+                (
+                    f"{self.batch_id}-item-submissions.csv",
+                    report.write_item_submissions_csv(StringIO()),
+                )
+            ],
         )
```

Comment:
I really like this method:
- `self` is a `Workflow`
- we get a configured `Report` instance by passing `self`
- we use the `SESClient` to send a message that we craft almost entirely from properties and methods